pre_commit_hooks/pretty_format_json.py

```diff
@@ -110,16 +110,16 @@ def pretty_format_json(argv=None):
             )
 
             if contents != pretty_contents:
-                print("File {} is not pretty-formatted".format(json_file))
+                print('File {} is not pretty-formatted'.format(json_file))
 
                 if args.autofix:
                     _autofix(json_file, pretty_contents)
 
                 status = 1
 
-        except simplejson.JSONDecodeError:
+        except json.JSONDecodeError:
```

Comment:
there's no `json.JSONDecodeError`, I believe stdlib throws `ValueError`?

Reply:
Ah it does exist on Py3 but not Py2... I'll sort this :+1: